src/test/java/htsjdk/samtools/util/IntervalCodecTest.java

```diff
@@ -59,27 +59,26 @@ public void testEndToEnd() throws IOException  {
         expectedList.add(new Interval("chr1", 50, 150, true, "number-5"));
         expectedList.add(new Interval("chr1", 150, 250, false, "number-6"));
 
-        final OutputStream outputStream = new FileOutputStream(tempFile);
-        final IntervalCodec writeCodec = new IntervalCodec(this.dict);
-        writeCodec.setOutputStream(outputStream);
-        for (final Interval interval : expectedList.getIntervals()) {
-            writeCodec.encode(interval);
+        try(final OutputStream outputStream = new FileOutputStream(tempFile)) {
+            final IntervalCodec writeCodec = new IntervalCodec(this.dict);
```

Comment:
need `this`?

Reply:
nope.  I removed it from the rest of the class too except where needed.